felix/bpf/tc/attach.go

```diff
@@ -659,7 +662,7 @@ func updateJumpMap(obj *libbpf.Obj, isHost bool, ipv6Enabled bool) error {
 		// in IPv6, we add the prologue program to the jump map, and the first entry is 4.
 		base := -1
 		if ipFamily == "IPv6" {
-			base = 4
+			base = 5
```

Comment:
Should we future proof by making the map say 32 in size and setting base=16?

Reply:
we will resolve this in ipv6 work